pip/build_manylinux.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Is to be executed in the Manylinux Docker container.
+# https://github.com/pypa/manylinux
+
+# Generates the manylinux .whl file.
+# Input:
+#   "/io" is mapped to this repo dir "qsharp" (from YAML, from repo dir  "docker run -v `pwd`:/io").
+#   Environment (from YAML `docker run --env ARCH=x86_64`):
+#       ARCH: Architecture: x86_64 or aarch64, etc.
+# Output:
+#   "<this repo>/target/wheels/qsharp-preview-*-cp3*-abi3-manylinux*_$ARCH.whl", 
+#       e.g., "<this repo>/target/wheels/qsharp-preview-0.0.7-cp37-abi3-manylinux_2_28_x86_64.whl".
+
+set -x -e
+
+# Install Rust:
+curl --proto '=https' --tlsv1.2 -sS https://sh.rustup.rs > rustup-init.sh
+chmod +x rustup-init.sh
```

Comment:
Would we be able to easily switch between msrustup and public rust here?

Reply:
We only can use msrustup in ado, which is where this is being used.